src/net.h

```diff
@@ -475,7 +475,7 @@ friend class CNode;
     */
     int64_t PoissonNextSendInbound(int64_t now, int average_interval_seconds);
 
-    void SetAsmap(std::vector<bool> asmap) { addrman.m_asmap = asmap; }
+    void SetAsmap(std::vector<bool> asmap) { addrman.m_asmap = std::move(asmap); }
 
```

Comment:
7cb3e60d5d5917f0a892271090e1297c7ddd722a seems to be missing a number of changes?

Reply:
Wrong commit mentioned in commit body, the body was for https://github.com/dashpay/dash/pull/4028/commits/9be6d8a887ace6575932e2473bd04d5b7dbf9e4f. https://github.com/bitcoin/bitcoin/commit/d58bcdc4b569a667b6974c3547b7ff6f665afce9's `init.cpp` fix is implemented in https://github.com/dashpay/dash/commit/36c368099a64a1dd888a1207600119e25adba856#diff-b1e19192258d83199d8adaa5ac31f067af98f63554bfdd679bd8e8073815e69dR1933-R1957

Reply:
So what commit is 74f271963f86fef885bb9a081b6f3d346b9865b7 actually backporting? I'm confused now

Reply:
It's supposed to be https://github.com/bitcoin/bitcoin/commit/d58bcdc4b569a667b6974c3547b7ff6f665afce9